debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p /mit
+ExecStartPre=/bin/chown root:pyhesiodfs ${pyhesiodfs_dir}
+ExecStartPre=/bin/chmod 770 ${pyhesiodfs_dir}
+ExecStart=/usr/bin/pyhesiodfs -f ${pyhesiodfs_dir} -o nonempty
+ExecStopPost=/bin/umount -l ${pyhesiodfs_dir}
```

Comment:
This `umount` fails because systemd has already stopped `pyhesiodfs` with `SIGTERM`. I think that’s sufficient and we should just get rid of this `ExecStopPost` line.

```
● debathena-pyhesiodfs.service - Hesiod automounter for Athena lockers
   Loaded: loaded (/lib/systemd/system/debathena-pyhesiodfs.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Wed 2015-05-13 00:20:10 EDT; 900ms ago
  Process: 29816 ExecStopPost=/bin/umount -l ${pyhesiodfs_dir} (code=exited, status=32)
 Main PID: 27418 (code=exited, status=0/SUCCESS)

May 13 00:19:44 fixed-disk systemd[1]: Starting Hesiod automounter for Athena lockers...
May 13 00:19:44 fixed-disk systemd[1]: Started Hesiod automounter for Athena lockers.
May 13 00:19:44 fixed-disk pyhesiodfs[27418]: Error: Invalid locker name: .Trash
May 13 00:19:44 fixed-disk pyhesiodfs[27418]: Error: Invalid locker name: .Trash-1000
May 13 00:20:10 fixed-disk systemd[1]: Stopping Hesiod automounter for Athena lockers...
May 13 00:20:10 fixed-disk umount[29816]: umount: /mit: not mounted
May 13 00:20:10 fixed-disk systemd[1]: debathena-pyhesiodfs.service: control process exited, code=exited status=32
May 13 00:20:10 fixed-disk systemd[1]: Stopped Hesiod automounter for Athena lockers.
May 13 00:20:10 fixed-disk systemd[1]: Unit debathena-pyhesiodfs.service entered failed state.
May 13 00:20:10 fixed-disk systemd[1]: debathena-pyhesiodfs.service failed.
```